tools/cldr-apps/pom.xml

```diff
@@ -18,7 +18,7 @@
 	<parent>
 		<groupId>org.unicode.cldr</groupId>
 		<artifactId>cldr-all</artifactId>
-		<version>38.0-SNAPSHOT</version>
+		<version>39.0-SNAPSHOT</version>
```

Comment:
How does the versioning work?

Reply:
Use of maven release plugin to drop snapshot, tag and bump a version is a popular way to do this.

Alternatively, we may have parent pom to manage single version across modules.

Reply:
you can't have a single version in the parent AND use the version for submodules to find the parent. 
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] 'parent.version' is missing. @ line 22, column 10
[FATAL] 'parent.version' is missing. @ line 18, column 10
```

Reply:
I think we can just bump the version whenever a new release is cut .  actually, when the maint branch is cut it can be manually updated.